Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -276,12 +276,12 @@ macro restoreCalleeSavesUsedByWasm()
 end
 
 macro preserveGPRsUsedByTailCall(gpr0, gpr1)
-    storep gpr0, CodeBlock[sp]
+    storep gpr0, ThisArgumentOffset[sp]
     storep gpr1, ArgumentCountIncludingThis[sp]
 end
 
 macro restoreGPRsUsedByTailCall(gpr0, gpr1)
-    loadp CodeBlock[sp], gpr0
+    loadp ThisArgumentOffset[sp], gpr0
```

Comment:
Is this change correct? Isn't it possible that this location gets clobbered during tail-call shuffling?